### DISCUSSION TITLE
GX650RX Screen not detected for refresh setting

### DISCUSSION THREAD

**Discussion by slashdelta** [seerge/g-helper#696] [Q&A]
I was having issues with G-Helper for a bit now, trying to troubleshoot as best I can.

Issue is:
I used to have the settings show for the screen refresh rate +OD, and the Multi/Single Zone backlighting option.
During an update of either G-Helper itself, or a GPU Driver update, that setting was now missing.

I have tried multiple different means to try and get it back, (Downgrade G-Helper, reinstall suggested drivers from G-Helper, latest versions of drivers, stop G-Helper and install ArmoryCrate) but to no avail.

Even in ArmoryCrate, the option to turn the backlighting from single to multi won't work.

 
I have just nuked my machine and reinstalled Windows from scratch, same symptoms.

[log.txt](https://github.com/seerge/g-helper/files/11900062/log.txt)
[config.json.txt](https://github.com/seerge/g-helper/files/11900063/config.json.txt)

Attached are the logs/config for the post-reinstall of Windows.

Perusing the log and config files, it seems that G-Helper isn't identifying the screen properly.
Don't know if that's a G-Helper Issue, or I borked something on my system.
Mind you, I just did a fresh install and the issue persists.

Any thoughts or sage words of wisdom?

Thanks,

slash

**Comment by seerge** [comment]
@slashdelta if you don't see a multizone switch in UI, it means that endpoint in bios responsible for it is missing :) (that's why AC also don't who it). I don't know why it happened with you, mb bios updates?

You can try to manually set multizone ON or OFF with the following command in PowerShell (1 for ON, 0 for OFF)

```
(Get-WmiObject -Namespace Root/WMI -Class AsusAtkWmi_WMNB).DEVS(0x0005001e, 1)
```

**Reply by slashdelta** [reply]
Hmmmmm,

Seems to be a BIOS update, but now I can't seem to get it working, even on an older BIOS version...

Thanks for the one-liner, didn't work, but good to have in the future

Thanks,

slash

**Reply by seerge** [reply]
Try to do a hardware reset https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop mb that will resurrect your miniled :)

**Reply by slashdelta** [reply]
Tried twice, didn't work...

I'm gonna try to flash a few older BIOS versions and hard reset them to see if that works.
If not, I guess I'm at the mercy of ASUS Support...

Side note, the external monitor I plug in shows as the laptop screen. Funny

Anyway, Thanks a ton for the support!

slash
